test/functional/mempool_accept.py

```diff
@@ -83,31 +83,31 @@ def run_test(self):
         )
 
         self.log.info('A transaction not in the mempool')
-        fee = 0.00000700
+        fee = Decimal('0.00000700')
         raw_tx_0 = node.signrawtransactionwithwallet(node.createrawtransaction(
             inputs=[{"txid": txid_in_block, "vout": 0, "sequence": BIP125_SEQUENCE_NUMBER}],  # RBF is used later
-            outputs=[{node.getnewaddress(): 0.3 - fee}],
+            outputs=[{node.getnewaddress(): Decimal('0.30000000') - fee}],
```

Comment:
Trailing zeros are ignored, so you can leave them out

Reply:
@MarcoFalke so should I remove trailing zeros for all or keep the things as they are right now after last commit in this PR?

Reply:
I'd prefer to remove them, unless there is a reason to add them